nni/experiment/config/common.py

```diff
@@ -66,6 +68,7 @@ class ExperimentConfig(ConfigBase):
     assessor: Optional[_AlgorithmConfig] = None
     advisor: Optional[_AlgorithmConfig] = None
     training_service: Union[TrainingServiceConfig, List[TrainingServiceConfig]]
+    _deprecated: Optional[str, Any] = None
```

Comment:
Deprecated fields as well as fields not supported by v2 (yet)...
It appears in at least 3 places and I placed the comments somewhere else. Because this file is "more visible" to end users.
This is a workaround for backward compatibility.